crates/bevy_render/src/mesh/primitives/dim3/plane.rs

```diff
@@ -7,39 +7,32 @@ use crate::{
 };
 
 /// A builder used for creating a [`Mesh`] with a [`Plane3d`] shape.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Default)]
 pub struct PlaneMeshBuilder {
```

Comment:
If this just wraps `Plane3d`, can't we remove it?

Reply:
Right now, we could, but we should probably add subdivisions back in a follow-up, so it'll be needed for that

Reply:
Sounds good.